public/templates/firebase.html

```diff
@@ -55,6 +58,26 @@
       type="text/css"
       href="/assets/css/owl-carousel.css"
     />
+
+    <!-- Scripts -->
+
+    <script defer src="/components/firebase/app.js"></script>
+    <!-- jQuery -->
+    <script defer src="/assets/js/jquery-2.1.0.min.js"></script>
```

Comment:
Hey,

I created an issue asking for the jquery to be updated (#68). You don't have to do the plugins but editing all the jquery stuff could be done in this pr. What do you think @j0n3lson?

Reply:
No, let's do the jQuery update in a separate PR so we have a small scope change for this PR (e.g. it's only about moving the script statements).

Let's leave this as is for now.